models/flower.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class Flower(models.Model):
+    _name = 'flower'
```

Comment:
Usually, in Odoo, when we define a new model, its technical name should be composed of at least two keywords, the first one being a **Domain** under which your custom models fall, and the second one being the actual name of your **Business Object**.

For example, in the **account** module, all model names start with the **account** keyword, as that is the domain name for that module. Ex: **account.move**, **account.tax**, **account.journal**, **account.account**

In this case, the model name should be **flower.flower**, where the first flower is the domain name, and the second is your business object.

Obviously, this is just a convention, and you do not have to adhere to it, but it is always good to be specific and consistent with the way Odoo does things.